EasyPost.Tests/ServicesTests/PartnerServiceTest.cs

```diff
@@ -0,0 +1,276 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using EasyPost.Exceptions.API;
+using EasyPost.Models.API;
+using EasyPost.Tests._Utilities;
+using EasyPost.Tests._Utilities.Annotations;
+using EasyPost.Utilities.Annotations;
+using RestSharp;
+using Xunit;
+
+namespace EasyPost.Tests.ServicesTests
+{
+    public class PartnerServiceTests : UnitTest
+    {
+        public PartnerServiceTests() : base("partner_service", TestUtils.ApiKey.Partner)
+        {
+        }
+
+        #region Tests
+
+        #region Test CRUD Operations
+
+        [Fact]
+        [CrudOperations.Create]
+        [Testing.Function]
+        public async Task TestCreateReferral()
+        {
+            UseVCR("create_referral");
+
+            ReferralCustomer referralCustomer = await CreateReferral();
+
+            Assert.NotNull(referralCustomer);
+            Assert.IsType<ReferralCustomer>(referralCustomer);
+            Assert.NotNull(referralCustomer.Id);
```

Comment:
We don't really need this since we have another similar assertion in the next line

Reply:
Resolved